src/__test__/components/Header/Header.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Header from '@components/Header';
+import { render } from '@testing-library/react';
```

Comment:
Please, create a order to import the stuff

- External libraries
- Shared components
- Inside components